api/src/main/resources/api/service_openapi.yaml

```diff
@@ -576,14 +583,128 @@ components:
           description: Repeated instances results of the search.
           nullable: true
           items:
-            $ref: '#/components/schemas/EntityInstanceStruct'
+            $ref: "#/components/schemas/EntityInstanceStruct"
         nextPageToken:
-          $ref: '#/components/schemas/NextPageToken'
+          $ref: "#/components/schemas/NextPageToken"
 
     EntityInstanceStruct:
       type: object
       description: A map of an entity instance attribute names to their value for this instance.
       additionalProperties:
         type: object
         nullable: true
-        $ref: '#/components/schemas/AttributeValue'
+        $ref: "#/components/schemas/AttributeValue"
+
+    UserData:
```

Comment:
Note: While cohort / conceptSet could (and probably should) be repurposed for a backend user artifact API, this UserData container seems likely to be specific to local storage indefinitely. May be worth a note in the description, but also won't matter too much until we have artifact APIs.

Reply:
I'll leave it for now. It'll either stay as is or migrate into the UI depending on how storage evolves.